File: Submissions/#0513/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    String S = sc.next();
    sc.close();

    String[] s = S.split("");

    int[] num = new int[10];

    for (int i = 0; i < n; i++) {
      int v = Integer.valueOf(s[i]);
      num[v]++; 
    }

    System.out.println(Math.min(n / 11, num[8]));
    
  }

}

File: Submissions/#0513/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n;
  cin >> n;

  string s;
  cin >> s;

  vector<int> num(10);

  for (int i = 0; i < n; i++) {
    int v = s[i] - '0';
    num[v]++; 
  }

  cout << min(n/11, num[8]) << endl;

}